chart/templates/core-agents-deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: core-agents
+  namespace: mayastor
+  labels:
+    app: core-agents
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: core-agents
+  template:
+    metadata:
+      labels:
+        app: core-agents
+    spec:
+      containers:
+        - name: core
+          image: {{ .Values.mayastorCP.registry }}mayadata/mcp-core:{{ .Values.mayastorCP.tag}}
```

Comment:
Should the `mayadata` portion be incorporated as part of the {{ .Values.mayastorCP.registry }}?

Reply:
I don't think so that part in invariant, on dockerhub the image name is `mayadata/mcp-core`

Reply:
Fair enough